src/utils/notifications.ts

```diff
@@ -72,6 +72,27 @@ export const handlePushManagerUnsubscribe = async (hide: () => void) => {
   }
 };
 
+export const manageNotification = async (accountId: string, notificationType: string, block: boolean) => {
+  const serviceWorker = await navigator.serviceWorker.ready;
+  const subscription = await serviceWorker.pushManager.getSubscription();
```

Comment:
I think that the check for null might be needed here just in case (as per the [docs](https://developer.mozilla.org/en-US/docs/Web/API/PushManager/getSubscription)).

Reply:
This function is called only when the user has turned on notification.

Added a condition for that and should be all fixed in the latest commit here:
https://github.com/near/near-discovery-components/pull/520